nifi-extension-bundles/nifi-kafka-bundle/nifi-kafka-processors/src/main/java/org/apache/nifi/kafka/processors/consumer/convert/AbstractRecordStreamKafkaMessageConverter.java

```diff
@@ -110,6 +110,8 @@ public void toFlowFiles(final ProcessSession session, final Iterator<ByteRecord>
                     processSingleRecord(session, recordGroups, consumerRecord, record, attributes, extraAttrs, topic, partition);
                 }
             } catch (final MalformedRecordException | IOException | SchemaNotFoundException e) {
+                logger.debug("Reader or writer failed to process Kafka message from topic {} partition {} offset {}",
```

Comment:
I recommend adjusting the message slightly, making it more generic since there are a number of possible causes:
```suggestion
                logger.debug("Failed to process Kafka Record with Topic [{}] Partition [{}] Offset [{}]",
```

Reply:
From the code it seems `MalformedRecordException`, `IOException` and `SchemaNotFoundException` can occur when interacting with reader/writer only.
I'm concerned that "Failed to process Kafka Record" might be too generic for the users, as it doesn't mention serde or IO operations.

The log format was adjusted.